godot-core/src/builtin/aabb.rs

```diff
@@ -82,6 +82,20 @@ impl Aabb {
     */
 }
 
+impl std::fmt::Display for Aabb {
+    /// Formats `Aabb` to match Godot's string representation.
+    ///
+    /// Example:
+    /// ```
```

Comment:
You applied the changes only here, not in all the other cases.

Reply:
Sorry, I will fix it :)